frontend/apps/app/features/projects/pages/ProjectsPage/ClientSearchWrapper.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+
+import { urlgen } from '@/utils/routes'
+import { ChevronDown } from '@liam-hq/ui'
+import Link from 'next/link'
+import type { FC } from 'react'
+import { ProjectItem, SearchInput } from '../../components'
+import { useProjectSearch } from '../../hooks/useProjectSearch'
+import styles from './ProjectsPage.module.css'
+
+interface Project {
+  id: number
+  name: string
+  createdAt: string
+  organizationId: number | null
+}
+
+interface ClientSearchWrapperProps {
+  initialProjects: Project[] | null
+  organizationId?: number
+}
```

Comment:
I think I can get the types from `supabase/database.types`.

```suggestion
import type { Tables } from '@liam-hq/db/supabase/database.types'
import { ChevronDown } from '@liam-hq/ui'
import Link from 'next/link'
import type { FC } from 'react'
import { ProjectItem, SearchInput } from '../../components'
import { useProjectSearch } from '../../hooks/useProjectSearch'
import styles from './ProjectsPage.module.css'

interface ClientSearchWrapperProps {
  initialProjects: Tables<'Project'>[] | null
  organizationId?: number
}
```